core/trino-main/src/main/java/io/trino/operator/OrderByOperator.java

```diff
@@ -220,7 +223,8 @@ public void finish()
                     // spill since revocable memory could not be converted to user memory immediately
                     // TODO: this should be asynchronous
                     getFutureValue(spillToDisk());
-                    finishMemoryRevoke.run();
```

Comment:
just call `finishMemoryRevoke`?

Reply:
finishMemoryRevoke is an API method; we;re relying on certain internals (for example, we don't call `startMemoryRevoking`), so it won't be more pretty if we call `finishMemoryRevoke`.